.github/release.yml

```diff
@@ -0,0 +1,28 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+    authors:
+      - dependabot
+  categories:
+    - title: Bug Fixes
+      labels:
+        - bug
+    - title: Enhancements
+      labels:
+        - enhancement
+    - title: LABEL_TITLE_1
```

Comment:
major? or is that implied with no label?

Reply:
We (or other maintainers) can add any label that is desired so long as we are staying on top of labelling issue and PRs consistently.